XLS-0070d-credentials/README.md

```diff
@@ -579,24 +569,28 @@ It's much faster to have the credential ID included - it's easy to make sure tha
 
 ### A.7: Can a credential issuer delete their account?
 
-Yes, even if they still have issued credentials in existence.
+Yes, though the credentials they created will be deleted.
 
 ### A.8: Does a credential issuer have to have an on-chain account?
 
-No, though they must have a valid keypair. If the second method of creating a credential is used (via a signature from the issuer), the issuer can still attest to a credential without having to have an account on-chain. This is to ensure closer compliance with the W3C Verifiable Credentials spec (see section 9).
+Yes.
 
 ### A.9: How do I get a list of credentials that an issuer has issued?
 
-The answer to that question is still being investigated. One option would be a Clio API.
+The `account_objects` RPC will return a list of all `Credential` objects an account is involved in - both as `Subject` and `Issuer`. To get a list of just the ones an account has issued, just filter that list for `Credential`s that have that account as the `Issuer`.
+
+### A.10: How do I get a list of credentials that an account has been issued?
+
+The `account_objects` RPC will return a list of all `Credential` objects an account is involved in - both as `Subject` and `Issuer`. To get a list of just the ones an account has been issued, just filter that list for `Credential`s that have that account as the `Subject`.
```

Comment:
> To get a list of just the ones an account has been issued, just filter that list for `Credential`s that have that account as the `Subject`.

Since we want to find `Credential`'s where the said account is the `Issuer`, the last sentence needs to re-phrased as below:
> To get a list of just the ones an account has been issued, just filter that list for `Credential`s that have that account as the `Issuer`.

Reply:
No, we want to find the credentials an account _has been issued_, not _has issued_.